datasets/ecommerce/selection.py

```diff
@@ -0,0 +1,33 @@
+
+
+files = open("/Users/kevin/project_java/COOL/datasets/ecommerce/update-sample.csv")
```

Comment:
add some annotations to explain the usage of this py file.

Reply:
Added